tests/optim/utils/test_reducer.py

```diff
@@ -18,7 +18,6 @@ def __init__(self, n_components=3, **kwargs) -> None:
         self.components_ = np.ones((2, 64))
 
     def fit_transform(self, x: torch.Tensor) -> np.ndarray:

```

Comment:
The type hint `np.ndarray` should be changed to `torch.Tensor` now that `x.numpy()` has been removed.